website/static/img/icons/148705-essential-collection/eps/add-1.eps

```diff
@@ -0,0 +1,82 @@
+%!PS-Adobe-3.0 EPSF-3.0
```

Comment:
what are `.eps` files? images? why can't they be jpeg?

Reply:
flaticon.com is where i got these icons, and they give you a directory with the license and the icons in different formats. I just copied the entire directory into single-spa.js.org. png, svg, and jpg versions are also available -- see https://github.com/CanopyTax/single-spa.js.org/tree/master/website/static/img/icons/148705-essential-collection